fsr-de/1327
# Revert view permission renaming

this is the second (and i think last) pr currently blocking updates on production.

in 69b67555c2ad we renamed view permissions. this was not reflected in respective changes in the database, resulting in all view permissions being lost after the migrations. it is actually not necessary to rename the permissions, we can just use the built-in view permission from django instead and everything continues working. so this pr reverts some changes from the above mentioned commit and makes sure to not redefine a custom view permission.

how to test:
1) have a vm running on the current `production` branch. when destroying and rebuilding the vm, make sure to change the box in the `Vagrantfile` to a newer version (e.g. like in #718), otherwise the installation will fail.
2) create some test documents, menu items, ... and assign some view permissions (ideally for a group with at least one non-superuser).
3) check out the branch from this pr, update the requirements and run all migrations.
4) the view permissions for the non-superuser should all still be there and working.
